template/pkg/{{Package}}/plugin.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) {{ Year }}, the Drone Plugins project authors.
+// Please see the AUTHORS file for details. All rights reserved.
+// Use of this source code is governed by an Apache 2.0 license that can be
+// found in the LICENSE file.
+
+package {{ Package }}
+
+import (
+	"github.com/drone-plugins/drone-plugin-lib/pkg/plugin"
+	"github.com/drone-plugins/drone-plugin-lib/pkg/urfave"
+)
+
+type pluginImpl struct {
```

Comment:
This naming is not idiomatic Go, simply name it plugin, and let's give the plugin lib another name.